applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -93,8 +94,10 @@ def PrepareModelPart(self):
                 KratosMultiphysics.Logger.PrintWarning(self.__class__.__name__, "Material properties have not been imported. Check \'material_import_settings\' in your ProjectParameters.json.")
             ## Replace default elements and conditions
             self._ReplaceElementsAndConditions()
-            ## Set buffer size
-            self.main_model_part.SetBufferSize(self.min_buffer_size)
+            ## Executes the check and prepare model process
+            self._ExecuteCheckAndPrepare()
+            ## Set and fill buffer
+            self._SetAndFillBuffer()
 
         ## Executes the check and prepare model process. Always executed as it also assigns neighbors which are not saved in a restart
         self._ExecuteCheckAndPrepare()
```

Comment:
We cab call `_ExecuteCheckAndPrepare` here and avoid calling it in line 98 ryt?

Reply:
Yes. Indeed I think that this accidentally sneaked in. Nice catch.